README.md

```diff
@@ -267,14 +271,40 @@ Authentication is also mandatory for any communication and ensure either by
 
 ## 8.3 Monitoring
 
+Monitoring is only required when running in Microservice mode. As the reconciler is deployed in KCP landscapes, the available Kyma observability stack in KCP will be used.
```

Comment:
```suggestion
Monitoring is only needed when the reconciler runs in microservice mode. Because the reconciler is deployed in KCP landscapes, the available Kyma observability stack in KCP is used.
```